VERSION_INFO

```diff
@@ -1 +1 @@
-0.1.0.dev
\ No newline at end of file
+0.1.0-rc.1
```

Comment:
Should this change mostly be in the release branch? We should keep the main branch 0.1.0.dev and once release branch has been made, we can change to 0.2.0.dev?

Reply:
0.1.0.dev is not a valid option for nuget.
 here is the list of valid options for nuget
 https://learn.microsoft.com/en-us/nuget/concepts/package-versioning?tabs=semver20sort

Reply:
ok, then shall we name it 0.1.0-dev? What I am trying to say is that the main branch should not have the rc tag in the version number.

Reply:
I think it will be better to be 0.1.0. As next office release number, without dev

Reply:
The problem with leaving the version as 0.1.0 is that there is a risk someone might use the main branch to publish (maybe accidentally), and we would want to avoid publishing to PyPI or to nuget from the main branch.